recipes/gnu-time/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "gnu-time" %}
+{% set version = "1.7" %}
+{% set sha256 = "e37ea79a253bf85a85ada2f7c632c14e481a5fd262a362f6f4fd58e68601496d" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh
```

Comment:
How is this synchronized? Wouldn't we need to require `automake` and pin it then?